release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3021,6 +3021,44 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+/ 4.18.16
+[id="ocp-4-18-16_{context}"]
+=== RHSA-2025:8284 - {product-title} {product-version}.16 bug fix update
+
+Issued: 03 June 2025
+
+{product-title} release {product-version}.16 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:8284[RHSA-2025:8284] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:8285[RHBA-2025:8285] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.18.16 --pullspecs
+----
+
+[id="ocp-4-18-16-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, in non-Zonal Azure regions, a bug in the dynamically computed fault domain count update caused scaling failures. This issue prevented upgrades to {product-title} version 4.15.48 and later from scaling existing machine sets. With this release, an update is implemented that rectifies the dynamic fault domain calculation and prevents scaling failures in non-Zonal regions. This fix ensures smooth operation of machine set scaling after upgrading to {product-title} version 4.15.48 and later. (link:https://issues.redhat.com/browse/OCPBUGS-56654[OCPBUGS-56654])
+
+* Previously, API server subject alternative name (SAN) validation was performed regardless of public key infrastructure (PKI) reconciliation status, causing potential connectivity issues with hosted control plane clusters due to invalid SANs. With this release, a fix removes validation for {product-title} API server SANs when PKI reconciliation is disabled. The hosted control plane performance is improved by eliminating unnecessary validation, especially when the PKI reconciliation is not managed. (link:https://issues.redhat.com/browse/OCPBUGS-56627[OCPBUGS-56627])
+
+* Previously, a bug in Bare Metal Operator (BMO) led to JSON parsing errors because of a missing Redfish system ID in Baseboard Management Controller (BMC) URLs. This issue caused users to receive errors when the system ID was left out of the URLs. With this release, BMO handles URLs without a Redfish system ID as addresses without a system ID. This fix improves the software's handling of missing a Redfish system ID in BMC URLs. (link:https://issues.redhat.com/browse/OCPBUGS-56431[OCPBUGS-56431])
```

Comment:
```suggestion
* Previously, a bug in the Bare Metal Operator (BMO) caused JSON parsing errors because of a missing Redfish system ID in Baseboard Management Controller (BMC) URLs. This issue caused users to receive errors when the system ID was left out of the URLs. With this release, BMO handles URLs without a Redfish system ID as addresses without a system ID. This fix improves software handling of missing a Redfish system ID in BMC URLs. (link:https://issues.redhat.com/browse/OCPBUGS-56431[OCPBUGS-56431])
```